api/nakadi-event-bus-api.yaml

```diff
@@ -170,6 +170,10 @@ paths:
           description: Server error
           schema:
             $ref: '#/definitions/Problem'
+        '503':
+          description: Service (temporary) not available
```

Comment:
I have an English question. For an English speaker, would it sound better "Service (temporarily) unavailable"?